acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/besu/BesuNode.java

```diff
@@ -52,12 +54,17 @@ public class BesuNode {
   private static final BigInteger SPURIOUS_DRAGON_HARD_FORK_BLOCK = BigInteger.valueOf(1);
 
   private final BesuNodeConfig besuNodeConfig;
+
+  @SuppressWarnings("unused")
   private final String[] args;
```

Comment:
It's used in this file so think we could remove this?

Reply:
removed